bundle/schema/embed_test.go

```diff
@@ -58,16 +58,6 @@ func TestJsonSchema(t *testing.T) {
 		assert.NotEmpty(t, pipeline.AnyOf[0].Properties[field].Description)
 	}
 
-	// Assert enum values are loaded
-	schedule := walk(s.Definitions, "github.com", "databricks", "databricks-sdk-go", "service", "pipelines.RestartWindow")
```

Comment:
There is no more enums inlined in the definition (the old way), so removing the test as we don't have anything to test against anymore.